content/docs/getting-started/overview/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Overview
+publishedDate: '2024-04-26T16:00:00.0Z'
+description: How to begin onboarding on to Roadie.
+---
+
+
+## Introduction
+
+Learn how to build your internal developer portal with Roadie:
+
+- Add users 
+- Populate your software catalog
+- Add plugins and customise your user interface
+- Dive into scorecards, templates and docs-as-code
+
+## 1. [Adding users](/docs/getting-started/user-management/)
```

Comment:
I think we would swap 1 and 2 around IMO. The end game is the catalog.. and we should get the catalog item in as fast as possible in my opinion. We can introduce ownership later as we discuss further in the documentation.

Reply:
Unfortunately, I think we need the current order for PoV prospects to know how to add their team before they start evaluating it. @samnixon87 , what do you think about merging `Model users, groups and teams` (missing an oxford comma?) and `Modelling your software` together, and making it a step later in the list?